package.json

```diff
@@ -8,7 +8,8 @@
     "url": "git://github.com/thlorenz/doctoc.git"
   },
   "scripts": {
-    "test": "set -e; for t in test/*.js; do node $t; done"
+    "test": "set -e; for t in test/*.js; do node $t; done",
+    "test:cov": "set -e; tap test/*.js --cov --coverage-report=lcov;"
```

Comment:
That's not part of this feature. Please remove from now.
You can propose test coverage in another PR and we can think about if we want to pull that.

Reply:
Should I rebase/squash to make those changes disappear or just add a new commit to undo those changes?